app/ui.js

```diff
@@ -1053,18 +1053,17 @@ var UI;
             }
         },
 
-        connect: function() {
+        connect: function(_, password) {
```

Comment:
This results in an error as well, if host or port is empty, since the name of the translation function is _:

 Uncaught TypeError: _ is not a function
    at HTMLDivElement.connect (ui.js:1078)